invenio_circulation/cli.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
```

Comment:
You should probably be fixing copyright headers on all files to something like:

```
Copyright (C) 2018 CERN
Copyright (C) 2018 RERO
```

Reply:
I hope it is done.

Reply:
wow! Next time, maybe better in a separate commit? Anyway, thanks for changing it.

Reply:
+1 for the separate commits especially when we touch a lot of files :)